### PR TITLE
Multilanguage

### PR DESCRIPTION
**Pull request by happyhuman**
Samples to demo the multi language capabilities

### CODE REVIEW

**Comment by happyhuman** [comment]
@nnegrey , @dizcology , @theacodes Can you please review this PR?

File: speech/cloud-client/transcribe_multilanguage.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        resources/Google_Gnome.wav
+    python transcribe_multilanguage.py \
+        gs://cloud-samples-tests/speech/Google_Gnome.wav
+"""
+
+import argparse
+
+
+# [START speech_transcribe_multilanguage]
+def speech_transcribe_multilang(speech_file):
```
Comment (nnegrey) [inline speech/cloud-client/transcribe_multilanguage.py]: speech_transcribe_multilanguage, thoughts?

File: speech/cloud-client/transcribe_multilanguage_test.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import re
+
+import transcribe_multilanguage
+
+RESOURCES = os.path.join(os.path.dirname(__file__), 'resources')
+OUTPUT1 = 'OK Google stream stranger things from Netflix to my TV'
```
Comment (nnegrey) [inline speech/cloud-client/transcribe_multilanguage_test.py]: Nit: OUTPUT

File: speech/cloud-client/transcribe_multilanguage.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+# limitations under the License.
+
+r"""Google Cloud Speech API sample that demonstrates how to request
+multiple languages.
+
+Example usage:
+    python transcribe_multilanguage.py \
+        resources/Google_Gnome.wav
+    python transcribe_multilanguage.py \
+        gs://cloud-samples-tests/speech/Google_Gnome.wav
```
Comment (nnegrey) [inline speech/cloud-client/transcribe_multilanguage.py]: This a public bucket? `cloud-samples-tests`

File: speech/cloud-client/README.rst [hunk 4]
```diff
@@ -230,6 +230,7 @@ To run this sample:
         python beta_snippets.py enhanced-model resources/commercial_mono.wav
         python beta_snippets.py metadata resources/commercial_mono.wav
         python beta_snippets.py punctuation resources/commercial_mono.wav
+        python beta_snippets.py multi-language resources/commercial_mono.wav
```
Comment (dizcology) [inline speech/cloud-client/README.rst]: this, and the changes to the beta_snippets.py file might cause conflicts against the other PRs.

File: speech/cloud-client/beta_snippets.py [hunk 5]
```diff
@@ -126,6 +127,35 @@ def transcribe_file_with_auto_punctuation(path):
...
+    with open(speech_file, 'rb') as audio_file:
+        content = audio_file.read()
+
+    audio = speech.types.RecognitionAudio(content=content)
+
+    config = speech.types.RecognitionConfig(
+        encoding=speech.enums.RecognitionConfig.AudioEncoding.LINEAR16,
+        sample_rate_hertz=16000,
+        language_code='ja-JP',
+        alternative_language_codes=['es-ES', 'en-US'])
```
Comment (dizcology) [inline speech/cloud-client/beta_snippets.py]: How does this work?  if the speech in the audio has multiple languages in it, say English and Japanese, then the transcription will have both?

File: speech/cloud-client/beta_snippets_test.py [hunk 6]
```diff
@@ -42,3 +44,11 @@ def test_transcribe_file_with_auto_punctuation(capsys):
     out, _ = capsys.readouterr()
 
     assert 'Okay. Sure.' in out
+
+
+def test_transcribe_multilanguage_file(capsys):
+    transcribe_file_with_multilanguage(
+        os.path.join(RESOURCES, 'Google_Gnome.wav'))
```
Comment (dizcology) [inline speech/cloud-client/beta_snippets_test.py]: Use a different test file that has multiple languages.

**Pull request closed without merging**